### PR TITLE
Ce/restore util error

### PR DESCRIPTION
**Pull request by calellowitz**
@benrudolph @sravfeyn cc: @snopoke 
wrote a test for https://github.com/dimagi/commcare-hq/pull/12421, found a second bug in the process

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/tests/test_restore_models.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase
+from corehq.apps.domain.models import Domain
+
+from corehq.apps.groups.models import Group
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/tests/test_restore_models.py]: F401 'Group' imported but unused

File: corehq/ex-submodules/casexml/apps/phone/tests/test_restore_models.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+from corehq.apps.users.models import CommCareUser, DomainMembership
+from corehq.apps.users.dbaccessors.all_commcare_users import delete_all_users
+
+DOMAIN = 'fixture-test'
+
+class OtaRestoreUserTest(TestCase):
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/tests/test_restore_models.py]: E302 expected 2 blank lines, found 1

**Comment by benrudolph** [comment]
👍 on ✔️

**Comment by benrudolph** [comment]
hooray for tests 😸

**Comment by calellowitz** [comment]
✅

**Comment by czue** [comment]
nice

**Pull request merged**